src/mem/alloc.h

```diff
@@ -45,6 +45,17 @@ namespace snmalloc
     PMNotOurs = 0,
     PMSuperslab = 1,
     PMMediumslab = 2
+
+    /*
+     * Values SUPERSLAB_BITS (inclusive) through 64 (exclusive, I suppse)
+     * are used for log2(size) at the heads of large allocations.
+     *
+     * Values 64 (inclusive) through 128 (exclusive) are used for entries
```

Comment:
The 64-128 are used for backwards jumps towards the start of an object.  They are the largest power of two that does not jump beyond the start.  I.e.  79 would be that there are at least `2^(79 - 64)` entries before this one to find the start of the object.  This allows logarithmic access to the start of the object.